Event_Display/ClusterShapeDisplay.cpp

```diff
@@ -0,0 +1,44 @@
+#include "TCanvas.h"
+#include "TH2D.h"
+#include "TStyle.h"
+#include <bitset>
+#include <iostream>
+
+void fillBinaryMatrix(TH2D *hist, const uint8_t *ExampleArray, int size,
+                      int max_length, int xClusterPosition,
+                      int yClusterPosition) {
+  for (int i = 0; i < size; ++i) {
+    std::bitset<8> bits(ExampleArray[i]);
+    for (int j = 0; j < max_length; j++) {
+      hist->Fill(xClusterPosition + j, yClusterPosition - i - 1,
+                 bits[max_length - 1 - j]); // Adjusting the y-position
+    }
+  }
+}
+
+void ClusterShapeDisplay() {
+  // need to link t cluster class
+  // uint8_t ExampleArray[] = {16, 48, 48, 224, 32, 0, 0, 0};// Example array
+  uint8_t ExampleArray[] = {192, 192, 80, 248, 16, 16, 16, 16};

```

Comment:
```suggestion
  std::array<uint8_t,MAX_CLUSTER_SIZE> ExampleArray = {192, 192, 80, 248, 16, 16, 16, 16};
```

Reply:
I have implemented it as an std::array in the end; could you try if ```MAX_CLUSTER_SIZE``` works?

Reply:
Thank you for your suggest, i will try in the Afternoon